src/main/java/org/openjdk/shenandoah/ShenandoahVisualizer.java

```diff
@@ -269,12 +264,63 @@ public void actionPerformed(ActionEvent e) {
             }
         };
         toolbarPanel.setResetSpeedListener(resetSpeedListener);
-
+        final int[] regionWidth = new int[1];
```

Comment:
I originally try to use just integer to declare it but somehow it can't build. Therefore, I took the suggestion from IntelliJ and uses array so it can actually build.